hazelcast/src/main/java/com/hazelcast/internal/util/MutableByte.java

```diff
@@ -16,7 +16,14 @@
 
 package com.hazelcast.internal.util;
 
-public class MutableByte {
+import com.hazelcast.nio.ObjectDataInput;
+import com.hazelcast.nio.ObjectDataOutput;
+import com.hazelcast.nio.serialization.DataSerializable;
+
+import java.io.IOException;
+import java.io.Serializable;
+
+public class MutableByte implements DataSerializable, Serializable {
```

Comment:
What requires that class to be Serializable?

Reply:
E.g, It's used wihin processor suppliers

Reply:
:(

Reply:
I removed that need, changed it to non-serializable